data/colors/deprecations.json

```diff
@@ -1 +1,397 @@
+  "mergeBox.neutralIndicatorBorder": null,
+  "mergeBox.warningIconBg": "attention.emphasis",
+  "mergeBox.warningIconText": "fg.onEmphasis",
+  "mergeBox.warningIconBorder": null,
+  "mergeBox.warningBoxBorder": "attention.emphasis",
+  "mergeBox.warningMergeHighlight": null,
+  "mergeBox.errorIconBg": "danger.emphasis",
+  "mergeBox.errorIconText": "fg.onEmphasis",
+  "mergeBox.errorIconBorder": null,
+  "mergeBox.errorIndicatorBg": "danger.emphasis",
+  "mergeBox.errorIndicatorBorder": null,
+  "underlinenav.border": null,
+  "underlinenav.borderHover": "neutral.muted",
+  "underlinenav.borderActive": "primer.border.active",
+  "underlinenav.text": "fg.default",
+  "underlinenav.textHover": "fg.default",
+  "underlinenav.textActive": "fg.default",
+  "underlinenav.icon": "fg.subtle",
+  "underlinenav.iconHover": "fg.subtle",
+  "underlinenav.iconActive": "fg.default",
+  "underlinenav.counterText": "fg.default",
+  "underlinenav.counterBg": "neutral.muted",
+  "selectMenu.borderSecondary": "border.muted",
+  "selectMenu.shadow": "shadow.large",
+  "selectMenu.backdropBg": "primer.canvas.backdrop",
+  "sidenav.borderActive": "primer.border.active",
+  "menu.headingText": "fg.default",
+  "menu.borderActive": "primer.border.active",
+  "project.cardBg": "canvas.overlay",
+  "prState.draft.text": "fg.onEmphasis",
+  "prState.draft.bg": "neutral.emphasis",
+  "prState.draft.border": null,
+  "prState.open.text": "fg.onEmphasis",
+  "prState.open.bg": "success.emphasis",
+  "prState.open.border": null,
+  "prState.merged.text": "fg.onEmphasis",
+  "prState.merged.bg": "done.emphasis",
+  "prState.merged.border": null,
+  "prState.closed.text": "fg.onEmphasis",
+  "prState.closed.bg": "danger.emphasis",
+  "prState.closed.border": null,
+  "diffBlob.numText": "fg.subtle",
+  "diffBlob.numHoverText": "fg.default",
+  "diffBlob.addition.numHoverText": "fg.default",
+  "diffBlob.deletion.numHoverText": "fg.default",
+  "diffBlob.deletion.lineBg": "danger.subtle",
+  "diffBlob.deletion.wordBg": "danger.muted",
+  "diffBlob.hunk.text": "fg.muted",
+  "diffBlob.hunk.numBg": "accent.muted",
+  "diffBlob.hunk.lineBg": "accent.subtle",
+  "diffBlob.emptyBlockBg": "neutral.subtle",
+  "diffBlob.selectedLineHighlightBg": "attention.subtle",
+  "diffBlob.selectedLineHighlightBorder": "attention.muted",
+  "diffBlob.expander.hoverIcon": "fg.onEmphasis",
+  "diffBlob.expander.hoverBg": "accent.emphasis",
+  "diffBlob.commentButton.icon": "fg.onEmphasis",
+  "diffBlob.commentButton.bg": "accent.emphasis",
+  "diffBlob.commentButton.gradientBg": null,
+  "globalNav.logo": null,
+  "globalNav.bg": null,
+  "globalNav.text": null,
+  "globalNav.icon": null,
+  "globalNav.inputBg": null,
+  "globalNav.inputBorder": null,
+  "globalNav.inputIcon": null,
+  "globalNav.inputPlaceholder": null,
+  "introShelf.gradientLeft": "accent.subtle",
+  "introShelf.gradientRight": "success.subtle",
+  "introShelf.gradientIn": "canvas.default",
+  "introShelf.gradientOut": null
+}
```

Comment:
Just a side thought, it would be awesome to automate this script and have actions check it in on prs. So nobody will have to remember to re-run the script.

Reply:
Yeah, I thought about that too. But I'm curious if that's something we always want to automate.

Reply:
Let's see how the manually keep the file up-to-date goes after the v2 migration, then we can re-evaluate.